tests/coalton-tests.lisp

```diff
@@ -10,6 +10,8 @@
 (defun %run-tests (name)
   (run-test-file (format nil "tests/test-files/~a" name)))
 
+;; (coalton-tests:run-test "tests/test-files/define-class.txt" 100)
```

Comment:
Is this just example use code? I'd say either take it out or flesh out the comment with 
```
;; usage example:
;; (coalton-tests...
```

Reply:
ack, a stray -- zapped. thank you.